### ISSUE TITLE
[Multimedia collage] Video button missing accessible name without image

### ISSUE THREAD

**Issue by svinkle** [Shopify/dawn#1160]
## Issue summary

When the multimedia collage section features a video without an image, the load `button` is missing an accessible name. The exclusion of an accessible name will not convey the purpose of the `button` to assistive technology users, leading to a confusiung user experience.

<details>
  <summary>Screenshots</summary>

  <img width="1058" alt="Multimedia collage, video is highlighted." src="https://user-images.githubusercontent.com/1392632/149035581-e10b0e31-ac25-4b84-bedb-a4fb405402a0.png">

</details>

#### HTML

```html
<button
  class="collage-content collage-card__image-wrapper media deferred-media__poster full-unstyled-link"
  type="button"
  aria-haspopup="dialog"
  data-media-id="_9VUPq3SxOc"
>
  <!-- SVG… -->
</button>
```

## Steps to reproduce

1. Go to a Dawn theme store with any supported browser
2. With an active screen reader, <kbd>Tab</kbd> through a Multimedia Collage section, video `button` withouth an image

### Expected

- `button` to include an accessible name to convey its purpose.

### Actual

- `button` missing an accessible name.

## Recommendation

Apply the `aria-label` attribute to the `button` element, setting its value to `Load video: {video.alt}`. With this, screen readers will announce the video description along with the callout, "Load video."

If the video does not include a description, the fallback of "Load video" will suffice as an accessible name.

#### HTML

```html
<button
  class="collage-content collage-card__image-wrapper media deferred-media__poster full-unstyled-link"
  type="button"
  aria-label="Load video: {{ video.alt }}"
  aria-haspopup="dialog"
  data-media-id="_9VUPq3SxOc"
>
  <!-- SVG… -->
</button>
```

<!-- #### Diff

```diff
- line removed
+ line added
``` -->

## Specifications

- **Browser:** Any
- **Dawn version:** 2.5.0
- **Component:** Multimedia collage
- **WCAG Principal:** Perceivable
- **WCAG SC:** [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- **Severity:** High
- **Effort:** Low
